backend/apps/volontulo/views/api.py

```diff
@@ -219,6 +219,18 @@ def get_queryset(self):
             )
         return qs.filter(offer_status='published')
 
+    @detail_route(methods=['POST'], permission_classes=(IsAuthenticated,))
+    # pylint: disable=invalid-name
+    def join(self, request, pk):
+        """Endpoint to join offer by current user"""
```

Comment:
"The docstring is a phrase ending in a period" (see: https://www.python.org/dev/peps/pep-0257/)

Reply:
True.

Reply:
So where is a period here?